search/elasticsearch/impl/pom.xml

```diff
@@ -79,6 +95,14 @@
             <version>0.2</version>
             <scope>test</scope>
         </dependency>
+        <!-- kernel-impl gives us ThreadLocalComponent, which is much easier to use for tests than is a mock
+         for that service, since interactions almost always assume a stateful implementation -->
+        <dependency>
+            <groupId>org.sakaiproject.kernel</groupId>
+            <artifactId>sakai-kernel-impl</artifactId>
+            <version>11-SNAPSHOT</version>
```

Comment:
This version tag should be removed since it's incorrect and should be inherited anyway.